wbstack/LocalSettings.php

```diff
@@ -70,13 +70,13 @@
 if( $wwDomainSaysLocal ) {
 	// TODO this code path shouldn't be accessible when in PROD
 	// TODO fix totally hardcoded port for dev us
-	$wgServer = "//" . $wikWiki->requestDomain . ":8083";
+	$wgServer = "https://" . $wikWiki->requestDomain . ":8083";
```

Comment:
These should perhaps be http:// actually as they are for localhost development.
I'll make a follow up, didn't spot until after merging!